commit 0d31b8e1dae2bd0ad73c90748a03f9cfeed837d8
Author: Olivier Houchard <ohouchard@haproxy.com>
Date:   Tue Oct 16 18:49:26 2018 +0200

    MINOR: peers: use defines instead of enums to appease clang.
    
    Clang (rightfully) warns that we're trying to set chars to values >= 128.
    Use defines with hex values instead of an enum to address this.
    
    (cherry picked from commit 33992267aac00d7e8ae67e0703bf7fffc9cf9b54)
    Signed-off-by: Willy Tarreau <w@1wt.eu>

diff --git a/src/peers.c b/src/peers.c
index 0cd56da3..465ffe85 100644
--- a/src/peers.c
+++ b/src/peers.c
@@ -122,15 +122,13 @@ enum {
 /* Note: ids >= 128 contains   */
 /* id message cotains data     */
 /*******************************/
-enum {
-	PEER_MSG_STKT_UPDATE = 128,
-	PEER_MSG_STKT_INCUPDATE,
-	PEER_MSG_STKT_DEFINE,
-	PEER_MSG_STKT_SWITCH,
-	PEER_MSG_STKT_ACK,
-	PEER_MSG_STKT_UPDATE_TIMED,
-	PEER_MSG_STKT_INCUPDATE_TIMED,
-};
+#define PEER_MSG_STKT_UPDATE           0x80
+#define PEER_MSG_STKT_INCUPDATE        0x81
+#define PEER_MSG_STKT_DEFINE           0x82
+#define PEER_MSG_STKT_SWITCH           0x83
+#define PEER_MSG_STKT_ACK              0x84
+#define PEER_MSG_STKT_UPDATE_TIMED     0x85
+#define PEER_MSG_STKT_INCUPDATE_TIMED  0x86
 
 /**********************************/
 /* Peer Session IO handler states */
